include/deal.II/matrix_free/matrix_free.h

```diff
@@ -1253,6 +1253,38 @@ class MatrixFree : public Subscriptor
        const DataAccessOnFaces src_vector_face_access =
          DataAccessOnFaces::unspecified) const;
 
+  template <typename CLASS, typename OutVector, typename InVector>
```

Comment:
Even though this functionality is not yet finished, I suggest we still add some basic documentation of the interface before merging.

Reply:
OK. Let me do it in the afternoon.

Reply:
I have copied the documentation from `loop()`, changed the introduction paragraph, and made some changes regarding the parameters (to reflect that there are no face loops).